google/cloud/storage/internal/grpc_client_test.cc

```diff
@@ -121,6 +122,12 @@ TEST_F(GrpcClientTest, DefaultOptionsGrpcEndpointNoEnv) {
   }
 }
 
+TEST_F(GrpcClientTest, DefaultOptionsUploadBuffer) {
```

Comment:
optional: `TEST(DefaultOptionsGrpc, UploadBuffer)`

and above with `EndpointNoEnv`, `ChannelCount`